### ISSUE TITLE
As AH unable to submit a plan (Chrome - laptop)

### ISSUE THREAD

**Issue by LisaMoore1** [bcgov/range-web#894]
After opening RUP (RAN099964) and selecting submit button following screen appears (undefined). Right side is below the yellow line is what was generated ![RAN099964 Submit (below yellow).jpg](https://images.zenhubusercontent.com/5d937161c8e4d80001227881/e115d122-eba1-4623-ad91-b1f2182fc127)

After doing a couple of other things I went back to Workque, opened same plan and saved and got this:
![RAN099964 Save.jpg](https://images.zenhubusercontent.com/5d937161c8e4d80001227881/5ad4421f-5165-4371-974d-3c5e7e94eb57)

**Comment by LisaMoore1** [comment]
Browser back button to return to WQ then reopened and saved: 
![return to WQ and save RAN099964.png](https://images.zenhubusercontent.com/5d937161c8e4d80001227881/87003500-c214-4656-a019-2d88510ad630)

**Comment by calebissharp** [comment]
@LisaMoore1 Did you reload the page after navigating back to the work queue?

**Comment by LisaMoore1** [comment]
login as andy
refresh WQ (using browser button)
Open RAN099964
Save draft
Get this:
![RAN099964 login refresh open save.jpg](https://images.zenhubusercontent.com/5d937161c8e4d80001227881/767359c1-f2c2-4fa0-b737-b3ca58a7dc47)

**Comment by LisaMoore1** [comment]
refresh WQ
Open RAN099964
Submit
![RAN099964 submit (below yellow)2.jpg](https://images.zenhubusercontent.com/5d937161c8e4d80001227881/a2afb868-d9a8-4602-90a2-76c44078bdd4)

**Comment by micheal-w-wells** [comment]
```
Error: There was a problem fetching the record. Error: select "plan"."id", "plan"."range_name", "plan"."plan_start_date", "plan"."plan_end_date", "plan"."notes", "plan"."alt_business_name", "plan"."agreement_id", "plan"."status_id", "plan"."uploaded", "plan"."amendment_type_id", "plan"."created_at", "plan"."updated_at", "plan"."effective_at", "plan"."submitted_at", "plan"."creator_id", "plan"."canonical_id", "plan"."is_restored", "plan"."conditions", "plan"."proposed_conditions", "ref_plan_status"."id" as "ref_plan_status_id", "ref_plan_status"."code" as "ref_plan_status_code", "ref_plan_status"."name" as "ref_plan_status_name", "ref_plan_status"."active" as "ref_plan_status_active", "extension"."id" as "extension_id", "extension"."request_date" as "extension_request_date", "extension"."end_date" as "extension_end_date", "extension"."notes" as "extension_notes", "user_account"."id" as "user_account_id", "user_account"."username" as "user_account_username", "user_account"."given_name" as "user_account_given_name", "user_account"."family_name" as "user_account_family_name", "user_account"."email" as "user_account_email", "user_account"."phone_number" as "user_account_phone_number", "user_account"."active" as "user_account_active", "user_account"."pia_seen" as "user_account_pia_seen", "user_account"."last_login_at" as "user_account_last_login_at" from "plan" inner join "ref_plan_status" on "plan"."status_id" = "ref_plan_status"."id" left join "extension" on "plan"."extension_id" = "extension"."id" inner join "user_account" on "plan"."creator_id" = "user_account"."id" where "plan"."id" = $1 and "uploaded" = $2 order by "id" desc - invalid input syntax for integer: "undefined"
--
  | at errorWithCode (/opt/app-root/src/node_modules/@bcgov/nodejs-common-utils/src/libs/utils.js:31:17)
  | at show (/opt/app-root/src/build/src/router/controllers_v1/PlanController.js:112:50)
  | at <anonymous>
```

@calebissharp

**Comment by LisaMoore1** [comment]
Still not working.  Wondering if it has anything to do with some weirdness in the minister’s issue (the not provided on the approved by minister) 
![8E820477-3404-49B8-BE44-26268D409329.png](https://images.zenhubusercontent.com/5d937161c8e4d80001227881/c10106c0-83ce-4dd8-b0e3-356b89046bfb)

Interesting because it is now not pulling any errors in dev tools but after I get the “Saved” message it is scrolling down to the MInister’s issue section of the plan (highlighting Minister’s Issues in the left hand menu). Might try making sure the approved by Minister info is correct and see if that fixes it.

**Comment by calebissharp** [comment]
I think this is a duplicate of #378.

**Comment by LisaMoore1** [comment]
Could be but the behaviour is different.  On this one I’m not getting the validation error messaging that I get for #378.  Related though I think. Maybe same issue but this one is the AH side versus what the staff get when they try to submit? I’m OK if you close this one as long as we make sure that the problem for RAN099964 (the only one I’ve noticed this with) is actually because of that toggle problem.

**Comment by calebissharp** [comment]
@LisaMoore1 Are you still getting this issue after #902 was merged in?

**Comment by LisaMoore1** [comment]
@calebissharp still not working.

**Comment by calebissharp** [comment]
@LisaMoore1 Are you still getting the same error as in this issue originally? ie. you get redirected to `/range-use-plan/undefined`?

**Comment by LisaMoore1** [comment]
@calebissharp I don't seem to be getting any now...it just isn't moving past the save part.

**Comment by calebissharp** [comment]
@LisaMoore1 I think this is because the minister issue toggle wasn't ever set to true.

**Comment by LisaMoore1** [comment]
@calebissharp can you or @micheal-w-wells manually change that to true and then we can see if that lets us push it through?

**Comment by calebissharp** [comment]
@micheal-w-wells would have to do it, I don't have DB access.

**Comment by LisaMoore1** [comment]
@micheal-w-wells would you mind switching the toggle to true for the minister’s issue on RAN099964 so we can see if we can clear this one off of here? Thx.

**Comment by LisaMoore1** [comment]
Switching the identified by minister to true solved the problem.  Closing.
